packages/synapse-react-client/src/utils/hooks/useDetectSSOCode.test.ts

```diff
@@ -49,23 +47,11 @@ const mockBindOAuthProviderToAccount = jest.spyOn(
   'bindOAuthProviderToAccount',
 )
 
-const AUTHENTICATED_CONTEXT: SynapseContextType = {
-  ...MOCK_CONTEXT_VALUE,
-}
-
-const UNAUTHENTICATED_CONTEXT: SynapseContextType = {
-  ...MOCK_CONTEXT_VALUE,
-  accessToken: undefined,
-}
-
 describe('useDetectSSOCode tests', () => {
```

Comment:
No longer relies on the ApplicationContext for the access token - it is given the token directly by the ApplicationSessionManager.